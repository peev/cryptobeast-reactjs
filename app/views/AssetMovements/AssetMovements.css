.AMGrid{
    flex-grow: 1;
    margin: 0;
}

.AMPaper{
    max-height:500px;
    width: 100%;
    padding: 10px;
    margin-top: 10px;
}
textarea{
    border-bottom: 2px solid black;
}

.AMPaper button{
    float:right;
}
