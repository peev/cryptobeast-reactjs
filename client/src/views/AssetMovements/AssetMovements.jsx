import React from 'react';
import { Grid } from 'material-ui';
import AssetInput from './../../components/Cards/Assets/AssetInput';
import AssetAllocation from './../../components/Cards/Assets/AssetAllocation';
import PortfolioSummaryTable from './../../components/CustomTables/PortfolioSummaryTable';

const AssetMovements = () => (
  <Grid container>
    <Grid item xs={12} sm={12} md={12}>
      <AssetInput />
    </Grid>
    <Grid item xs={12} sm={12} md={12}>
      <AssetAllocation />
    </Grid>
    <Grid item xs={12} sm={12} md={12}>
      <PortfolioSummaryTable
        tableHead={[
          'Ticker',
          'Holdings',
          'Price (BTC)',
          'Price (USD)',
          'Total Value (USD)',
          'Asset Weight',
          '24H Change',
          '7D Change',
        ]}
      />
    </Grid>
  </Grid>
);

export default AssetMovements;
