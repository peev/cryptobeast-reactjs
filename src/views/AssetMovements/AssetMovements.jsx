import React from 'react';
import { Grid } from 'material-ui';
import AssetInput from '../../components/Cards/Assets/AssetInput';
import AssetAllocation from '../../components/Cards/Assets/AssetAllocation';
import AssetsTable from '../../components/CustomTables/AssetsTable';

class AssetMovements extends React.Component {
  state = {
    direction: 'row',
  };

  render() {
    return (
      <div>
        <Grid container>
          <AssetInput />

          <AssetAllocation />

          <AssetsTable />
        </Grid>
      </div>
    );
  }
}

export default AssetMovements;
