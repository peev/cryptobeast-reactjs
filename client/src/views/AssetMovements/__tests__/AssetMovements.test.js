import React from 'react';
import { shallow } from 'enzyme';
import AssetMovements from './../AssetMovements';

describe('Views/AssetMovements.jsx', () => {
  const assetMovements = shallow(<AssetMovements />);

  it('should render without crashing', () => {
    expect(assetMovements).toMatchSnapshot();
  });
});
