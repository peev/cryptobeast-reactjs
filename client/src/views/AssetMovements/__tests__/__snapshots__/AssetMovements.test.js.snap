// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Views/AssetMovements.jsx should render without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AssetMovements />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <WithStyles(GridWrapper)
          item={true}
          md={12}
          sm={12}
          xs={12}
>
          <WithStyles(inject-AssetInput-with-AssetStore-MarketStore-PortfolioStore-NotificationStore) />
</WithStyles(GridWrapper)>,
        <WithStyles(GridWrapper)
          item={true}
          md={12}
          sm={12}
          xs={12}
>
          <WithStyles(inject-AssetAllocation-with-AssetStore-NotificationStore-MarketStore-PortfolioStore) />
</WithStyles(GridWrapper)>,
        <WithStyles(GridWrapper)
          item={true}
          md={12}
          sm={12}
          xs={12}
>
          <WithStyles(inject-_class-with-PortfolioStore)
                    tableHead={
                              Array [
                                        "Ticker",
                                        "Holdings",
                                        "Price(BTC)",
                                        "Price(USD)",
                                        "Total Value(USD)",
                                        "Asset Weight",
                                        "24H Change",
                                        "7D Change",
                                      ]
                    }
          />
</WithStyles(GridWrapper)>,
      ],
      "container": true,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(inject-AssetInput-with-AssetStore-MarketStore-PortfolioStore-NotificationStore) />,
          "item": true,
          "md": 12,
          "sm": 12,
          "xs": 12,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(inject-AssetAllocation-with-AssetStore-NotificationStore-MarketStore-PortfolioStore) />,
          "item": true,
          "md": 12,
          "sm": 12,
          "xs": 12,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(inject-_class-with-PortfolioStore)
            tableHead={
                        Array [
                                    "Ticker",
                                    "Holdings",
                                    "Price(BTC)",
                                    "Price(USD)",
                                    "Total Value(USD)",
                                    "Asset Weight",
                                    "24H Change",
                                    "7D Change",
                                  ]
            }
/>,
          "item": true,
          "md": 12,
          "sm": 12,
          "xs": 12,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "tableHead": Array [
              "Ticker",
              "Holdings",
              "Price(BTC)",
              "Price(USD)",
              "Total Value(USD)",
              "Asset Weight",
              "24H Change",
              "7D Change",
            ],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <WithStyles(GridWrapper)
            item={true}
            md={12}
            sm={12}
            xs={12}
>
            <WithStyles(inject-AssetInput-with-AssetStore-MarketStore-PortfolioStore-NotificationStore) />
</WithStyles(GridWrapper)>,
          <WithStyles(GridWrapper)
            item={true}
            md={12}
            sm={12}
            xs={12}
>
            <WithStyles(inject-AssetAllocation-with-AssetStore-NotificationStore-MarketStore-PortfolioStore) />
</WithStyles(GridWrapper)>,
          <WithStyles(GridWrapper)
            item={true}
            md={12}
            sm={12}
            xs={12}
>
            <WithStyles(inject-_class-with-PortfolioStore)
                        tableHead={
                                    Array [
                                                "Ticker",
                                                "Holdings",
                                                "Price(BTC)",
                                                "Price(USD)",
                                                "Total Value(USD)",
                                                "Asset Weight",
                                                "24H Change",
                                                "7D Change",
                                              ]
                        }
            />
</WithStyles(GridWrapper)>,
        ],
        "container": true,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <WithStyles(inject-AssetInput-with-AssetStore-MarketStore-PortfolioStore-NotificationStore) />,
            "item": true,
            "md": 12,
            "sm": 12,
            "xs": 12,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <WithStyles(inject-AssetAllocation-with-AssetStore-NotificationStore-MarketStore-PortfolioStore) />,
            "item": true,
            "md": 12,
            "sm": 12,
            "xs": 12,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <WithStyles(inject-_class-with-PortfolioStore)
              tableHead={
                            Array [
                                          "Ticker",
                                          "Holdings",
                                          "Price(BTC)",
                                          "Price(USD)",
                                          "Total Value(USD)",
                                          "Asset Weight",
                                          "24H Change",
                                          "7D Change",
                                        ]
              }
/>,
            "item": true,
            "md": 12,
            "sm": 12,
            "xs": 12,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "tableHead": Array [
                "Ticker",
                "Holdings",
                "Price(BTC)",
                "Price(USD)",
                "Total Value(USD)",
                "Asset Weight",
                "24H Change",
                "7D Change",
              ],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
